src/Scenario/InfoClientScenario.php

```diff
@@ -79,4 +82,22 @@ protected function pollAndTerminate(float $timeout = 0.0, bool $terminate = true
             $this->infoClient->terminate();
         }
     }
+
+    protected function getGenerator(): ClientGeneratorInterface
+    {
+        return $this->generator;
+    }
+
+    /**
+     * Connect to telegram with info (second) account
+     *
+     * @throws TGException
+     */
+    protected function infoLogin(): void
+    {
+        $authKey = AuthKeyCreator::createFromString($this->getGenerator()->getAuthKeyInfo());
+        if (!$this->infoClient->isLoggedIn()) {
+            $this->infoClient->login($authKey, $this->getGenerator()->getProxy());
+        }
+    }
```

Comment:
Should not we get rid of it?

Reply:
right, appeared after rebase, removed now.